authorizations/build.gradle.kts

```diff
@@ -17,6 +17,18 @@
  * under the License.
  */
 
-tasks.all {
-    enabled = false
-}
\ No newline at end of file
+tasks {
+  test {
+    subprojects.forEach {
+      dependsOn(":${project.name}:${it.name}:test")
+    }
+  }
+
+  register("copyLibAndConfig", Copy::class) {
+    subprojects.forEach {
+      if (!it.name.startsWith("authorization-common")) {
+        dependsOn(":${project.name}:${it.name}:copyLibAndConfig")
+      }
+    }
+  }
+}
```

Comment:
Do we need these codes? I found that each module already has these tasks.

Reply:
We can use it to optimize `compileDistribution` in the root `build.gradle.kts`.